script.module.resolveurl/lib/resolveurl/plugins/tusfiles.py

```diff
@@ -42,7 +42,7 @@ def get_media_url(self, host, media_id):
             'method_premium': ''
         }
         resp = self.net.http_POST(web_url, form_data=payload, headers=headers).get_url()
-        if resp:
+        if resp and resp != web_url:
```

Comment:
Actually resp will always exist due to previous line. so this line can be
if resp != web_url:

Reply:
Cool deal. Done